packages/create-cloudflare/templates/svelte/templates/wrangler.toml

```diff
@@ -1,85 +0,0 @@
-#:schema node_modules/wrangler/config-schema.json
-name = "<TBD>"
-compatibility_date = "<TBD>"
-pages_build_output_dir = ".svelte-kit/cloudflare"
-
-# Automatically place your workloads in an optimal location to minimize latency.
```

Comment:
same as https://github.com/cloudflare/workers-sdk/pull/7676/files#r1904327259, here we're dropping like most of the file's content, which is probably not what we want? 😕 

cc. to @IgorMinar and @jculvey who I think iterated over a few times in the past

Reply:
I missed the conversation above, it sounds like the removal of these comments was decided and intentional